tests/test_main.py

```diff
@@ -97,7 +97,7 @@
         ),
     ],
 )
-def test_dictionary_add(items: list, pairs_after_adding: list):
+def test_dictionary_add(items: list, pairs_after_adding: list[set]) -> None:
```

Comment:
The type hint for `pairs_after_adding` is incorrect. It should be `list[tuple]` instead of `list[set]`, as `pairs_after_adding` is a list of tuples.